src/api/contact-form.ts

```diff
@@ -0,0 +1,101 @@
+import { WebClient, LogLevel } from '@slack/web-api';
+
+const TOKEN = process.env.SLACK_BOT_SY_TOKEN;
+
+/**
+ * Easily build this block structure via https://app.slack.com/block-kit-builder
```

Comment:
really nice tool I found after I wasted too much time in the documentation
![Screen Shot 2022-01-06 at 07 20 44](https://user-images.githubusercontent.com/1701755/148337976-58cacc15-2e75-403b-9260-0b102eeea7da.png)

Reply:
Looks very practical 💫 Nice that there is such a tool